src/common/regions/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { concat, partition } from 'lodash';
+import regions from './region_db';
+import { getStateFips } from './regions_data';
+import { County, State, Region, MetroArea } from './types';
+
+export function belongsToState(county: County, stateFips: string | null) {
+  return county.state.fipsCode === stateFips;
+}
+
+export function getAutocompleteLocations(region?: Region) {
+  const { states, metroAreas, counties } = regions;
+
+  // Homepage
+  if (!region) {
+    return concat<Region>(states, metroAreas, counties);
```

Comment:
we want to include USA and NAMC here right?

Reply:
No, they don't have location pages - I'm adding them back on the autocomplete for explore